lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -161,6 +161,22 @@ def test_savefig_metadata(monkeypatch):
     }
 
 
+def test_invalid_metadata(monkeypatch):
+    fig, ax = plt.subplots()
+
+    with pytest.warns(UserWarning,
```

Comment:
Since you're here, could you add a case for an unknown keyword (triggering 213)?

Reply:
Good idea.